hexrd/fitting/grains.py

```diff
@@ -234,7 +233,7 @@ def objFuncFitGrain(gFit, gFull, gFlag,
         gHat_c = mutil.unitVector(np.dot(rMat_c.T, gVec_s))
 
         # !!!: check that this operates on UNWARPED xy
-        match_omes, calc_omes = matchOmegas(
+        _match_omes, calc_omes = matchOmegas(
```

Comment:
If `_match_omes` is not being used, this should be replaced with `_`. Though, it is a bit strange that `matchOmegas` would be called, and the `_match_omes` result would not be used. Something smells off in the workflow.